turbo.json

```diff
@@ -2,7 +2,13 @@
     "$schema": "https://turbo.build/schema.json",
     "pipeline": {
       "build": {
-        "outputs": ["dist/**", "target/**"],
+        "inputs": ["src/**"],
+        "outputs": ["target/**", "dist/**"],
+        "cache": false
+      },
+      "build:extension": {
+        "inputs": ["src/**"],
+        "outputs": ["target/**", "dist/**"],
         "cache": false
       },
```

Comment:
why two different pipeline but with the same parameters ?

Reply:
Because the pipeline does not compile the same things
The build one builds everything (also libs on their own) meanwhile the build:extension builds only what is leeded for the extension to run/be published

Reply:
Turborepo requirement